### DISCUSSION TITLE
Suggestion - Add menu item description in bootstrap_5_wp_nav_menu_walker

### DISCUSSION THREAD

**Discussion by binvoyons** [bootscore/bootscore#606] [Ideas]
Hi! Recently, I had to add the description of Wordpress native menu items in the nav walker, I think this should be included by default in bootScore, to avoid having to create a new walker. Just a thought! :)

`    $item_output = $args->before;
    $item_output .= '<a' . $attributes . '>';
    $item_output .= $args->link_before . apply_filters('the_title', $item->title, $item->ID) . $args->link_after;
	  	  if (!empty($item->description)) {
            $item_output .= '<span class="d-block menu-item-description">' . esc_html($item->description) . '</span>';
        }
    $item_output .= '</a>';
    $item_output .= $args->after;`

**Comment by crftwrk** [comment]
Thanks. I think it's more sustainable to direct this to https://github.com/AlexWebLab/bootstrap-5-wordpress-navbar-walker.
